rust/stackablectl/src/constants.rs

```diff
@@ -2,6 +2,8 @@ pub const ENV_KEY_RELEASE_FILES: &str = "STACKABLE_RELEASE_FILES";
 pub const ENV_KEY_STACK_FILES: &str = "STACKABLE_STACK_FILES";
 pub const ENV_KEY_DEMO_FILES: &str = "STACKABLE_DEMO_FILES";
 
+pub const DEMOS_REPOSITORY_URL_BASE: &str = "https://raw.githubusercontent.com/stackabletech/demos";
```

Comment:
question: Are the other constants (like `REMOTE_DEMO_FILE` and similar) used somewhere else. If so, it might cause problems based on the changes this PR introduces.

Reply:
note: It might be good to add at least two new constants, containing `demos/demos-v2.yaml` and `stacks/stacks-v2.yaml` and using them constructing the URL.

Reply:
`REMOTE_DEMO_FILE` and `REMOTE_STACK_FILE` are not used elsewhere in this or other repos and should will removed.

Reply:
Done in e6f73ca.